sql/core/benchmarks/AggregateBenchmark-jdk11-results.txt

```diff
@@ -0,0 +1,143 @@
+================================================================================================
+aggregate without grouping
+================================================================================================
+
+OpenJDK 64-Bit Server VM 11.0.4+11-LTS on Linux 3.10.0-862.3.2.el7.x86_64
+Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+agg w/o group:                            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+agg w/o group wholestage off                      55644          59484         NaN         37.7          26.5       1.0X
```

Comment:
`NaN` Stdev? Is it constantly `NaN`?

Reply:
Hmm. I'm not sure, but it's the generated result of the current benchmark suite.
It exists in the master branch from your commit https://github.com/apache/spark/commit/93a264d05a55c2617d34e977dbaf182987187a27 , too.

Reply:
This would suggest only 1 trial was run on this one. Not wrong per se but ideally these benchmarks all run a few iterations.

Reply:
Oh, right.